visualization-requirements.typ

```diff
@@ -29,7 +29,7 @@ The load balancer will be showcased at the innovation exposition in April. To he
     [#req("Must function as a useful debugging tool", mustHave)],
   ),
   caption: "General Visualization Requirements"
-)
+) <Figure::visualization-requirements>
```

Comment:
nit: I don't really like this using the `Figure::` prefix but we don't really have an established naming convention for labeling requirements tables